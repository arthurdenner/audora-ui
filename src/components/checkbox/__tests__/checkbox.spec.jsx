import React from 'react';
import { render } from 'enzyme';
import Checkbox from '../checkbox';

describe('Checkbox', () => {
  it('renders correctly', () => {
    const component = (<Checkbox label="Test Checkbox" name="testCheckbox" value={1} />);
    const wrapper = render(component);
    expect(wrapper).toMatchSnapshot();
  });
});
