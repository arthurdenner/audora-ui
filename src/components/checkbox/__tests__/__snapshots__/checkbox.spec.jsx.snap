// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders correctly 1`] = `
<label
  class="sc-bdVaJa hKAetY"
>
  <input
    name="testCheckbox"
    type="checkbox"
    value="1"
  />
  <span
    class="sc-bwzfXH drzoWa"
  >
    Test Checkbox
  </span>
</label>
`;
